src/components/custom-field-input-text/__snapshots__/custom-field-input-text.test.jsx.snap

```diff
@@ -0,0 +1,34 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CustomFieldInputText has defaults 1`] = `
+<div
+  className="custom-field-input-text"
+>
+  <div
+    className="heading-container"
+  >
+    <label
+      className="label"
+    >
+      Input Descriptor
+    </label>
+    <span
+      className="additional-information-icon"
```

Comment:
As previously discussed, we should be removing this.

Reply:
My bad, forgot to update the snapshot!